dashboard/client/src/service/job.ts

```diff
@@ -8,3 +8,8 @@ export const getJobList = () => {
 export const getJobDetail = (id: string) => {
   return get<JobDetailRsp>(`jobs/${id}`);
 };
+
+export const getJobProgress = (jobId?: string) => {
+  const jobIdQuery = jobId ? `?job_id=${jobId}` : "";
+  return get<JobProgressRsp>(`api/progress${jobIdQuery}`);
```

Comment:
Does it return null if success=False?

Reply:
fixed to add an error message in the UI.